### DISCUSSION TITLE
How to setup Memos behind Apache reverse-proxy?

### DISCUSSION THREAD

**Discussion by LilCharlie** [usememos/memos#3660] [Q&A]
Hi, when I tried to use Memos behind Apache reverse-proxy, I am only getting a blank page. Following is the entry for Memos in my proxy configuration file:
```
ProxyPass /memos/  http://192.168.0.22:8081/
ProxyPassReverse /memos/  http://192.168.0.22:8081/
<Location /memos/>
Order allow,deny
Allow from all
</Location>
```
So when I type, for example,` https://mydomain/memos/` (of course not giving my real domain name here), all I get is a blank page. This configuration works for most of my services, but not for Memos. Is there anything else can be done to serve memos through my secure domain without needing to open ports?

**Comment by TheNomad11** [comment]
My reverse proxy looks like this, very simple and basic:

```
<VirtualHost *:80>
  ServerName memos.domain.com
  ProxyPreserveHost On
  ProxyPass / http://127.0.0.1:5330/
  ProxyPassReverse / http://127.0.0.1:5330/
</VirtualHost>
```
Then I get a certificate via certbot that automatically makes the necessary changes for https

**Reply by LilCharlie** [reply]
Thanks
